packages/browser-utils/src/metrics/browserMetrics.ts

```diff
@@ -129,12 +129,12 @@ export function startTrackingLongAnimationFrames(): void {
   // we directly observe `long-animation-frame` events instead of through the web-vitals
   // `observe` helper function.
   const observer = new PerformanceObserver(list => {
+    if (!getActiveSpan()) {
+      return;
+    }
     for (const entry of list.getEntries() as PerformanceLongAnimationFrameTiming[]) {
-      if (!getActiveSpan()) {
-        return;
-      }
       if (!entry.scripts[0]) {
-        return;
+        continue;
```

Comment:
correct me if I'm missing something but this is still a behavior change. The `return` statement before would cause an early return of the entire callback while `break` or `continue` only influence the loop execution. 

Since you didn't update the PR description: Please explain why you think we should change behaviour here.